theme_real_estate/static/src/scss/primary_variables.scss

```diff
@@ -118,7 +134,4 @@ $o-color-palettes-compatibility-indexes: (
 // Shapes
 //------------------------------------------------------------------------------
 
-$o-bg-shapes: change-shape-colors-mapping('web_editor', 'Origins/14', (4: 4));
```

Comment:
Keep it for compatibility with snippets already dropped?
It will have an impact if the colors have not been edited by the user.